Lib/fontbakery/profiles/universal.py

```diff
@@ -571,32 +573,32 @@ def com_google_fonts_check_required_tables(ttFont, config):
                        "BASE", "GPOS", "GSUB", "JSTF",
                        "gasp", "hdmx", "LTSH", "PCLT",
                        "VDMX", "vhea", "vmtx", "kern"]
-    # See https://github.com/googlefonts/fontbakery/issues/617
-    #
-    # We should collect the rationale behind the need for each of the
-    # required tables above. Perhaps split it into individual checks
-    # with the correspondent rationales for each subset of required tables.
-    #
-    # com.google.fonts/check/kern_table is a good example of a separate
-    # check for a specific table providing a detailed description of
-    # the rationale behind it.
```

Comment:
This comment is still a valid TODO/FIXME. Please do not remove it.

Reply:
These all seemed outdated/superseded by the new information; but OK.

Reply:
Reinstated in 7a650fa7526a1884ff3462836289d94bad063c34